v1/nix/modules/flake-parts/apps.update-locks.nix

```diff
@@ -0,0 +1,48 @@
+# custom app to update the eval-cache of each exported package.
+{
+  self,
+  lib,
+  inputs,
+  ...
+}: {
+  perSystem = {
+    config,
+    self',
+    inputs',
+    pkgs,
+    system,
+    ...
+  }: let
+    l = lib // builtins;
+
+    allNewFileCommands =
```

Comment:
maybe just `refreshScripts`?

Reply:
fixed in 88b056bf6a72b9f996faf0b85cc6cd9392b1512f